### DISCUSSION TITLE
Plotting stuck for days at 100%

### DISCUSSION THREAD

**Discussion by ihaveanawesomeusername** [Chia-Network/chia-blockchain#3451] [Support]
Plotting is moving at normal speed until it hits 100% (about 10 hours to reach 100%) then it is stuck for days at 100%. It is still doing something as I can see the size of the .tmp file increasing in the destination folder but at an incredibly slow rate (maybe a MB every hour). Happening every single time I plot with:
Chia version 1.1.2 
MacBook Pro running Big Sur 11.3

Have tried restarting the application, different hard drives, different SSDs. Farming and syncing are working normally.

**Comment by sargonas** [comment]
At 100% it's trying to move the temp files over to the final destination location for farming.. what is that destination? Is it another internal drive, the same drive you plotted on, or an external one.. and if external what type?

**Reply by ihaveanawesomeusername** [reply]
Yes trying to move to an external drive. Have tried moving to multiple different brand and size external HDDs from different locations. I am never plotting on these drives and am plotting on an internal SSD and an external SSD to the same results.

**Reply by sargonas** [reply]
Understood... these are over USB I presume? Is it USB-C or USB A.. actually to make it easier, what model MBP are you using? I'm wondering if there might be some bottlenecking on the data transfer rates on your external connection. I can look up what the specs are for you given MBP probable and find out if you have a particularly restrictive one maybe, if it's an older model.

**Reply by ihaveanawesomeusername** [reply]
Thanks for the quick responses. Multiple USB A to multiple type WD drives (Elements, Passport). Just to be clear this never happened on my first 27 plots wit the same connection and is now happening every time.

**Reply by sargonas** [reply]
Well... that's not what I expected.... hmmm I don't have a lot of ideas off the top of my head myself, let me see if others have thoughts.

**Reply by ihaveanawesomeusername** [reply]
Ok thanks. In the meantime, I quit again (20 hours into 100%) and am installing Chia 1.1.3 and then adding a new plot, should take me about 10 hours to get to the 100% mark this time and I'll see if it happens with the new release. I didn't see any fix for this issue in the release notes but thinking it's worth updating and trying again while I'm waiting.

**Reply by ajantoniou** [reply]
Same issue with Mac OS and external passport drive for final directory. stuck at 100%. Log says "only read 0 of 262143 bytes at offset 3932145 from "temporary" with length 301973976. retrying in 5 minutes." And continues retrying for days. I have the latest 2TB sandisk temporary storage. When I open the temporary directory I see all .tmp files. After it pushed them through, for some reason it has a hard time reading them back from temporary to final directory.

**Comment by moshenskyDV** [comment]
Could you please provide plotter log? (three dots on the right of plot)

**Reply by kuzzenator** [reply]
Hello, I've got the similar problem.
Im plotting oninternal SATA SSD and my final folder is stored on external iSCSI storage.
The plotting is freezed at 100%.
Both Temp and Final folders got the same .TMP file with the equal size.
Logs in the attach. [Logs.txt](https://github.com/Chia-Network/chia-blockchain/files/6457247/Logs.txt)
![2021-05-11_10-15-28](https://user-images.githubusercontent.com/84006213/117774564-55ae3d00-b242-11eb-9b4a-6bcfffbe1485.png)

**Reply by moshenskyDV** [reply]
@kuzzenator just wait until it finish moving file to the final directory.

**Reply by lissizza** [reply]
Have the same problem, waiting more than 12 hours, is it normal?

**Reply by moshenskyDV** [reply]
@lissizza could you open task manager and check disk bandwidth? Are disk busy?
Next question: are you moving only one plot per moment of time, or you do a couple of plots at same moment (as @kuzzenator ), in this case process will be prolongated because of concurrency.

**Reply by kuzzenator** [reply]
Well i killed all other processes expect 2 plots from local SATA SSD to iSCSI LUN. Since the final folder is stored on multiple disks in RAID, im assuming that the bottleneck is network between 2 devices. If I'll add new network link will the plotting process use it without restarting the OS?

**Reply by linc600** [reply]
having exact same issue, not sure what to do now, see temp files in both temp (local SSD) and final location (NAS)

**Comment by Teledyol** [comment]
I had similar problem Running macOS 11.4 (beta) the plot log showed the dis was full.

I have a 32T HDD but the plots supposedly totaled 29.102 TiB

The disk utility also shows that I have full disk.

I think I'll drob those plots that won, or look for other issues. But the disk is full

**Reply by moshenskyDV** [reply]
Don't replot winners plots!

**Reply by Teledyol** [reply]
I saw that, but I think I want a little more free space in the drive, so I think that dropping those is perferrable

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
